### PR TITLE
Feat: Add Fuse UI and Changes for Fusenet

### PR DESCRIPTION
**Pull request by mul53**
- Added Fuse UI
- Added Changes for Fusenet

### CODE REVIEW

File: src/apollo/client.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { HttpLink } from 'apollo-link-http'
 
 export const client = new ApolloClient({
   link: new HttpLink({
-    uri: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
+    uri: 'http://3.22.187.59/subgraphs/name/fuseio/fuseswap',
```
Comment (mul53) [inline src/apollo/client.js:7]: Should i change these to env params?

File: src/apollo/queries.js [hunk 2]
```diff
@@ -734,17 +734,6 @@ export const TOKEN_CHART = gql`
       dailyVolumeETH
       dailyVolumeToken
       dailyVolumeUSD
-      mostLiquidPairs {
```
Comment (mul53) [inline src/apollo/queries.js:737]: This attribute is not declared for the type in the schema which was causing, maybe its dynamic or they just didn't update the repo with the new schema

File: yarn.lock [hunk 3]
```diff
@@ -1993,10 +1993,10 @@
     "@typescript-eslint/types" "4.4.0"
     eslint-visitor-keys "^2.0.0"
 
-"@uniswap/token-lists@^1.0.0-beta.15":
-  version "1.0.0-beta.15"
-  resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.15.tgz#93d2e9b2b2d2cfb0ce7fb4d30fffb685efde4256"
-  integrity sha512-cxubml6fPCjPPr8tWnA3IQkYbTocsZTJWac2C5WRXoHiSd7O4xjuBISdGLT9dW7IojKYpMXnCPJS8kAwqqYXDQ==
+"@uniswap/token-lists@^1.0.0-beta.19":
```
Comment (leonprou) [inline yarn.lock:1996]: do you need our token list here?

File: src/apollo/client.js (hunk 1 above)
Comment (leonprou) [reply src/apollo/client.js:7]: Don't think we need to introduce more changes here. These values should be are used only once here, so it'll be easy to change them

File: yarn.lock (hunk 3 above)
Comment (mul53) [reply yarn.lock:1996]: That package is more of a utility for working with tokenlists
The description below is from the repo
`This package includes a JSON schema for token lists, and TypeScript utilities for working with token lists.`

File: src/apollo/client.js (hunk 1 above)
Comment (mul53) [reply src/apollo/client.js:7]: Cool

File: src/apollo/queries.js (hunk 2 above)
Comment (leonprou) [reply src/apollo/queries.js:737]: Weird, does it exists here https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2 ?
Anyway seems like a feature we don't need currently

**Review by leonprou — APPROVED** [review]
Looks good.
Answered your questions, please see mine. We can merge after that

File: src/apollo/queries.js (hunk 2 above)
Comment (mul53) [reply src/apollo/queries.js:737]: Yes it exists, it was preventing the charts from rendering since it was returning an error, i will try to see why

**Pull request merged**